.github/workflows/dockerimage.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
     - name: 'Checkout Github Action'
-      uses: actions/checkout@master
+      uses: actions/checkout@Handlebars
```

Comment:
It's might be better to keep align with the case of branch name, i.e. `handlebars`.